src/transformers/models/vit/modeling_vit.py

```diff
@@ -126,7 +126,7 @@ def forward(
     ) -> torch.Tensor:
         batch_size, num_channels, height, width = pixel_values.shape
         embeddings = self.patch_embeddings(pixel_values, interpolate_pos_encoding=interpolate_pos_encoding)
-
```

Comment:
Any reason this file gets updated?

Reply:
Just leftover code from another PR, I undid all changes to ViT